microceph/client/remote.go

```diff
@@ -0,0 +1,62 @@
+package client
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/canonical/lxd/shared/api"
+	"github.com/canonical/lxd/shared/logger"
+	"github.com/canonical/microceph/microceph/api/types"
+	microCli "github.com/canonical/microcluster/client"
+	"github.com/canonical/microcluster/state"
+)
+
+// Sends the desired list of services to be restarted on every other member of the cluster.
+func SendRemoteImportRequest(ctx context.Context, c *microCli.Client, data types.Remote) error {
```

Comment:
Nit: if this function is about restarting services suggest to have this reflected in the function name

Reply:
ack